_includes/section-sidebar-menu.html

```diff
@@ -5,6 +5,7 @@
     <h2>Menu</h2>
     <p>{{ locale.project.description }}</p>
   </header>
+  <span>Elixir {{site.elixir.version}} Erlang/OTP {{site.erlang.otp}} [erts-{{site.erlang.erts}}]</span>
```

Comment:
I don't see the OTP version on [the screenshot](https://github.com/elixirschool/elixirschool/pull/1409#issuecomment-384401062). Most likely the reason is case-sensitivity of the config keys. Shouldn't it be `{{site.erlang.OTP}}` instead?

Reply:
![image](https://user-images.githubusercontent.com/73386/39278433-615988b6-48b0-11e8-81a3-c92311299ff0.png)